hieradata/role/jenkins-sandbox.yaml

```diff
@@ -9,6 +9,33 @@ psick::jenkins::scm_sync_repository_url: git@github.com:graduway/jenkins-config-
 psick::jenkins::scm_sync_repository_host: github.com
 
 ### SSH Key for SCM Sync
+psick::openssh::configs_hash:
```

Comment:
This looks like a copy and paste. If it's the same between different environments, it should be in one file.